.github/workflows/nightly.yml

```diff
@@ -883,13 +883,13 @@ jobs:
       matrix:
         include: ${{ fromJson(needs.GENERATE_MATRIX.outputs.windows-matrix-include) }}
     name: "${{ matrix.branch.name }}_WINDOWS_${{ matrix.x64 && 'X64' || 'X86' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}"
-    runs-on: windows-2022
+    runs-on: ${{ matrix.runner }}
```

Comment:
I'd prefer checking the version here, like we already do in some places:

https://github.com/php/php-src/blob/027b210d4253acac54dc18389eb6ca05ee5d62a5/.github/workflows/nightly.yml#L73

Reply:
A more reliable check would be `(matrix.branch.version.minor >= 3 || matrix.branch.version.major > 8)`, I suppose (without doing the full `((matrix.branch.version.major == 8 && matrix.branch.version.minor >= 3) || matrix.branch.version.major > 8)`

Reply:
Ah, thank you! Makes perfect sense for consistency. I went with the simple minor comparison; when the time for PHP 9.0, this could be updated together with the rest.